robot_assignments/web/jet2.robot

```diff
@@ -0,0 +1,97 @@
+*** Settings ***
+Library     SeleniumLibrary
+Library     String
+
+*** Variables ***
+${urlLogin}    http://125.26.15.143:13132/ 
+${urlhome}      http://125.26.15.143:13132/home.html
+${username1}    helloworld
+${password1}    1234
+${username2}    oz4899
+${password2}    1234
+${loginIDFieldXpath}    //input[@id='username']
+${loginPasswordFieldXpath}      //input[@id='password']
+${loginButtonXpath}     //button[@type='submit']
+${homePageContentXpath}     //div[@class='container']/div[@class='box']
+${searchFieldXpath}     //div[@class='column is-four-fifths']/input 
+${searchButtonXpath}       //button[@id='searchbtn'] 
+${itemListXpath}        //div[@class='card-content p-4']/p
```

Comment:
is it has id? please avoid using class as it contain like ' p-4' which easy to be changed.